hazelcast/src/main/java/com/hazelcast/sql/SqlColumnType.java

```diff
@@ -83,6 +84,7 @@ public enum SqlColumnType {
     JSON(15, HazelcastJsonValue.class),
 
     /** ROW type, represented by {@link RowValue} */
```

Comment:
Can we add small documentation saying that this column type is experimental, and could change or be removed in future releases without prior notice? I think we can't really remove beta annotated things, rather change them slightly, so I wanted to leave more slack here